.github/workflows/ci.yaml

```diff
@@ -141,6 +141,10 @@ jobs:
               ],
           )
           EOF
+
+      - name: Setup buildx
```

Comment:
I don't understand why this is needed. The genrule runs the same thing as part of the build, right?

Reply:
No, this creates a new driver in local daemon to allow `--builder=container` argument in genrule.  And we need a driver that supports exporting an oci-layout, instead of requiring us to call docker save to get the built image off of the daemon.

Reply:
okay so needs a comment then. Also the `||true` is suprpsing, it's okay for this to fail?

Reply:
|| true might not be necessary. removing.

Reply:
added the comment.